src/Todoist.Net/Serialization/Converters/CommandResultConverter.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+using Todoist.Net.Models;
+
+namespace Todoist.Net.Serialization.Converters
+{
+    internal class CommandResultConverter : JsonConverter<CommandResult>
+    {
+        public override CommandResult Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            if (reader.TokenType == JsonTokenType.String)
+            {
+                if (reader.GetString() != "ok")
```

Comment:
Let's use case insensitive comparison here

Reply:
Ok.

Reply:
The new comparison:

```c#
if (!string.Equals(reader.GetString(), "ok", StringComparison.OrdinalIgnoreCase))
```